src/style.css

```
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: auto;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > :first-child {
```

```suggestion
.stars--1 > :first-child,
.stars--2 > :nth-child(-n + 2),
... {
```
Consider using commas to group selectors and avoid repeating code